services/static-webserver/client/source/class/osparc/vipMarket/AnatomicalModelDetails.js

```diff
@@ -89,56 +89,61 @@ qx.Class.define("osparc.vipMarket.AnatomicalModelDetails", {
       if (modelsInfo.length > 1) {
         const modelSelectionLayout = new qx.ui.container.Composite(new qx.ui.layout.VBox(4));
         const titleLabel = new qx.ui.basic.Label(this.tr("This bundle contains:"));
         modelSelectionLayout.add(titleLabel);
-        const thumbnailsLayout = new qx.ui.container.Composite(new qx.ui.layout.HBox(2));
-        modelSelectionLayout.add(thumbnailsLayout);
+        const modelsLayout = new qx.ui.container.Composite(new qx.ui.layout.VBox(4));
+        modelSelectionLayout.add(modelsLayout);
         const thumbnailTapped = idx => {
           this.__populateSelectedModelInfo(idx);
           const selectedBorderColor = qx.theme.manager.Color.getInstance().resolve("strong-main");
           const unselectedBorderColor = "transparent";
-          thumbnailsLayout.getChildren().forEach((thumbnail, index) => {
+          modelsLayout.getChildren().forEach((thumbnailAndTitle, index) => {
+            const thumbnail = thumbnailAndTitle.getChildren()[0];
             osparc.utils.Utils.updateBorderColor(thumbnail, index === idx ? selectedBorderColor : unselectedBorderColor);
           });
         }
         modelsInfo.forEach((modelInfo, idx) => {
+          const modelLayout = new qx.ui.container.Composite(new qx.ui.layout.HBox(4));
           const miniThumbnail = this.self().createThumbnail(modelInfo["source"]["thumbnail"], 32);
-          miniThumbnail.set({
-            toolTipText: osparc.store.LicensedItems.licensedResourceNameAndVersion(modelInfo),
-          });
           osparc.utils.Utils.addBorder(miniThumbnail);
+          modelLayout.add(miniThumbnail);
           miniThumbnail.addListener("tap", () => thumbnailTapped(idx));
-          thumbnailsLayout.add(miniThumbnail);
+          const title = new qx.ui.basic.Label().set({
+            value: osparc.store.LicensedItems.licensedResourceTitle(modelInfo),
+            alignY: "middle"
+          });
+          modelLayout.add(title);
+          modelsLayout.add(modelLayout);
         });
-        modelLayout.add(modelSelectionLayout);
+        modelBundleLayout.add(modelSelectionLayout);
         thumbnailTapped(0);
 
         this.__populateSelectedModelInfo();
       } else {
         this.__populateSelectedModelInfo();
       }
 
-      this._add(modelLayout);
+      this._add(modelBundleLayout);
     },
 
     __populateSelectedModelInfo: function(selectedIdx = 0) {
       this.__selectedModelLayout.removeAll();
 
-      const anatomicalModelsData = this.getAnatomicalModelsData();
+      const licensedItemBundleData = this.getAnatomicalModelsData();
 
       const topGrid = new qx.ui.layout.Grid(8, 6);
       topGrid.setColumnFlex(0, 1);
       const headerLayout = new qx.ui.container.Composite(topGrid);
-      const anatomicalModel = anatomicalModelsData["licensedResources"][selectedIdx]["source"];
+      const anatomicalModel = licensedItemBundleData["licensedResources"][selectedIdx]["source"];
```

Comment:
there seem to be quite a lot of magic strings. would it make sense to define these somewhere (e.g. at top of file) and use those key variables?

Reply:
Once the data itself establishes, I'll create a proper model on my side 👍